app/frontend/components/domains/requirement-template/screens/edit-requirement-template-screen/sections-display.tsx

```diff
@@ -24,7 +24,17 @@ export const SectionsDisplay = observer(function SectionsDisplay(props: IProps)
   const watchedSections = watch("requirementTemplateSectionsAttributes")
 
   return (
-    <Stack w={"full"} alignItems={"flex-start"} spacing={16} p={16}>
+    <Stack
+      w={"full"}
+      alignItems={"flex-start"}
+      spacing={16}
+      mt="8"
+      mb="20"
+      mx="auto"
+      pl="8"
+      pr="130px" // space for floating buttons
```

Comment:
can this be a variable?

also see: app/frontend/components/shared/permit-applications/requirement-form.tsx